src/api/protectedApiClient.ts

```diff
@@ -165,6 +228,123 @@ const changePrivilegeLevel = (request: {
     .catch((e) => e);
 };
 
+const createTeam = (request: {
+  name: string;
+  bio: string;
+  inviteEmails: string[];
```

Comment:
⛏️ This is the same type as on line 42, maybe we might want to declare an interface for this?

Reply:
yes, something like `CreateTeamRequest`?